renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "local>Ocramius/.github:renovate-config"
```

Comment:
Note: I try to maintain this file to the best of my maintenance requirements, which are currently quite aligned with @lcobucci's, but we may need to import the whole DSL here, if preferred.

The advantage is more control, the disadvantage is that it's a lot of changes over time :D

Reply:
Ah, I just realized that renovate doesn't sign commits :thinking: 

Probably need to enable that...

Reply:
Other detail that I forgot: this action will create PRs until we fix the BC check action (currently failing at every PR, because `5.0.0` is unreleased and full of BC breaks)

Reply:
That's awesome, @Ocramius!
Thanks :+1: 

> Ah, I just realized that renovate doesn't sign commits thinking
> Probably need to enable that...

We can drop the signature requirements if they can't handle it.

> Other detail that I forgot: this action will create PRs until we fix the BC check action (currently failing at every PR, because 5.0.0 is unreleased and full of BC breaks)

It's failing because of some changes that use a syntax that seems to be unsupported. I'll ignore them for now: https://github.com/lcobucci/jwt/pull/967/commits/fb0597538c353c1ed2f48068a51d827a02232cb6

Reply:
I've just tested and renovate is signing commits... I've copied over your template to a repo under my control.

I'll update this one and we can merge it.